admin/templates/base.html

```diff
@@ -0,0 +1,109 @@
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body class="skin-blue sidebar-mini">
+    <div class="wrapper">
+
+      <!-- Main Header -->
+      <header class="main-header">
+
+        <!-- Logo -->
+        <a href="{% url 'home' %}" class="logo">
+          <!-- mini logo for sidebar mini 50x50 pixels -->
+          <span class="logo-mini"><b>O</b>SF</span>
+          <!-- logo for regular state and mobile devices -->
+          <span class="logo-lg"><b>OSF</b>Admin</span>
+        </a>
+
+        <!-- Header Navbar -->
+        <nav class="navbar navbar-static-top" role="navigation">
+          <!-- Navbar Right Menu -->
+          <div class="navbar-custom-menu">
+
+            <ul class="nav navbar-nav">
+              <!-- User Account Menu -->
+              <li class="dropdown user user-menu">
+                <a href"#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                  <span class="hidden-xs">Logged in as {{ user.get_username }}</span>
+                </a>
+              </li>
+              <li>
+                <a href="/logout/" class="btn btn-danger btn-flat">Sign out</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <!-- Left side column. contains the logo and sidebar -->
+      <aside class="main-sidebar">
+
+        <!-- sidebar: style can be found in sidebar.less -->
+        <section class="sidebar">
+          <!-- Sidebar Menu -->
+          <ul class="sidebar-menu">
+            <li class="header">Menu</li>
+            <li><a href="{% url 'spam:spam' %}"><i class='fa fa-link'></i> <span>OSF Spam</span></a></li>
+          </ul><!-- /.sidebar-menu -->
+        </section>
+        <!-- /.sidebar -->
+      </aside>
+
+      <!-- Content Wrapper. Contains page content -->
+      <div class="content-wrapper">
+        <!-- Content Header (Page header) -->
+
+        <!-- Main content -->
+        <section class="content">
+          {% block content %}
+          <h1>
+           Welcome to the OSF Admin Home Page
+          </h1>
+          <!-- Your Page Content Here -->
+          {% endblock %}
+        </section><!-- /.content -->
+      </div><!-- /.content-wrapper -->
+
+      <!-- Main Footer -->
+      <footer class="main-footer">
+        <!-- Default to the left -->
+        <strong>Copyright &copy; 2015 <a href="http://www.osf.io">osf.io</a>.</strong> All rights reserved.
+      </footer>
+
+    </div><!-- ./wrapper -->
+
+    <!-- REQUIRED JS SCRIPTS -->
+
+    <!-- Bootstrap 3.3.2 JS -->
+    {% render_bundle 'vendor' %}
```

Comment:
Probably worth declaring a `bottom_js` block or something similar here. Since this is a "layout" other things will be inheriting it and eventually will want to place javascript at the bottom of the page and, potentially, the top of the page. Better to define that now. Give them a convention to follow like the one in OSF. 

Or think about building a common base layout that both the OSF and OSF admin share. No reason this has to have any markup or styling in common. It could just be a bunch of blocks. The child templates could inherit and render those blocks inside their own html.

Just an idea...